docs/design.rst

```diff
@@ -15,37 +12,43 @@ by the Driver.
 API Access
 --------------------------
 
-Some Metagov endpoints are public, and others are restricted to **local traffic** ("INTERNAL"). Internal endpoints can only be accessed by the :ref:`Driver`.
-For now, the public endpoints are completely public (there is no auth).
+All drivers (that is, both Django and HTTP Drivers) make use of the API endpoints provided by Metagov to communicate with third-party platforms. These public endpoints allow platforms like Slack, Github, etc, to notify Metagov (and therefore the driver) when actions happen on those platforms.
 
+Additionally, HTTP Drivers must use the API to access Metagov's other functionality (whereas Django Drivers can directly import and call functions, models, etc). To enable this, a separate app called HTTPWrapper wraps Metagov's functionality into HTTP endpoints. To access these endpoints, Drivers need to make Driver Accounts on a local or hosted system and then authenticate to those endpoints.
 
-+----------------------------------------------+----------+--------------------------------------------+
-|                 Metagov URL                  |  Access  |                Description                 |
-+==============================================+==========+============================================+
-| ``/api/hooks/<community>/<plugin>``          | PUBLIC   | Receive incoming events from external      |
-|                                              |          | platforms (Open Collective, Loomio, etc)   |
-+----------------------------------------------+----------+--------------------------------------------+
-| ``/api/internal/community/<community>``      | INTERNAL | Create/update/delete a community           |
-+----------------------------------------------+----------+--------------------------------------------+
-| ``/api/action/<plugin>.<action>``            | PUBLIC   | Perform an action that is configured       |
-|                                              |          | to be public by the plugin author          |
-+----------------------------------------------+----------+--------------------------------------------+
-| ``/api/internal/action/<plugin>.<action>``   | INTERNAL | Perform an action                          |
-+----------------------------------------------+----------+--------------------------------------------+
-| ``/api/internal/process/<plugin>.<process>`` | INTERNAL | Perform an asynchronous governance process |
-+----------------------------------------------+----------+--------------------------------------------+
+.. WARNING::
 
+    The functionality described in this section is in development. Please reach out for help when working in this area.
+
+
+.. list-table:: Metagov APIs
+   :widths: 40 21 39
+   :header-rows: 1
+
+   * - Metagov URL
+     - Used By
+     - Description
+   * - ``/api/hooks/<community>/<plugin>``
+     - ALL
+     - Receive incoming events from external platforms (Open Collective, Loomio, etc)
```

Comment:
For Django Drivers, its up to them how to construct this URL. So it doesn't need to take that exact form, necessarily. I don't think we need to say that in this table, since its kinda niche, but just pointing it out. For example in PolicyKit, the hook URLs look like this:
https://github.com/amyxzhang/policykit/blob/d2884b6b70d6e21e894cc0f2e8774e729be148b0/policykit/policykit/urls.py#L66-L69